bolt/test/dynamic-relocs.test

```diff
@@ -0,0 +1,13 @@
+// This test checks that BOLT creates entry points for addresses
+// referenced by dynamic relocations.
```

Comment:
Since the test directly checks the computed goto code, I would rather keep the "indirect goto" part in the test name and in the description.

Reply:
Renamed and added this to the test description.